feat(components): add toolbar component (#350)
Vonage/vivid-3: libs/components/src/lib/action-group/README.md

> # Action-group

Author: ```suggestion
# Action Group
```

> An action group is a grouping of action buttons or form elements that are related to each other.

Author: ```suggestion
Action group visually groups buttons or form elements, related to each other, in a scoped region.
Note: this element have no semantic representation under its hood.
```

```js
<script type="module">import '@vonage/vivid/action-group';</script>
```

```html preview
<vwc-action-group>
  <vwc-button icon="reply-line"></vwc-button>
  <vwc-button label="copy"></vwc-button>
  <vwc-button label="paste"></vwc-button>
  <vwc-button label="submit"></vwc-button>
</vwc-action-group>
```

## Properties

### Appearance
Set the `appearance` attribute to change the action-Group's appearance.

> - Type: `'ghost'` | `'fieldset'`

Author: ```suggestion
- Type: `'fieldset'` | `'ghost'`
```

  <vwc-button label="paste"></vwc-button>
  <vwc-button label="submit"></vwc-button>
</vwc-action-group>
<vwc-action-group appearance="ghost">
  <vwc-button label="edit" appearance="filled"></vwc-button>
  <vwc-button label="copy" appearance="filled"></vwc-button>
  <vwc-button label="paste" appearance="filled"></vwc-button>
  <vwc-button label="submit" appearance="filled"></vwc-button>
</vwc-action-group>
```

### Shape
Use the `shape` attribute to set the action-Group's edges.  
When using shape - pay in mind setting the slotted elements with the same shape property.

- Type: `'rounded'` | `'pill'`
- Default: `'rounded'`

```html preview
<vwc-action-group shape="pill">
  <vwc-button label="edit" shape="pill"></vwc-button>
  <vwc-button label="copy" shape="pill"></vwc-button>
  <vwc-button label="paste" shape="pill"></vwc-button>
  <vwc-button label="submit" shape="pill"></vwc-button>
</vwc-action-group>
```

### Separator
Use the `<hr>` tag for adding separator between the action elements 

```html preview
<vwc-action-group appearance="fieldset">
  <vwc-button icon="reply-line"></vwc-button>
  <vwc-button icon="transfer-line"></vwc-button>

>   <hr>

Author: revisiting this topic -
as `hr` stands for horizontal rule, it implicitly sets its orientation to horizontal.
we should consider having a [dedicated divider component](https://explore.fast.design/components/fast-divider) just for the sake of a11y concern
alternatively, we can apply style to any element with a `role="separator"`.

see
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role

Reviewer: Changed the `<hr>` to span with `role="separator"`
Opened an issue regarding a [divider component](https://github.com/Vonage/vivid-3/issues/407)

Author: excellent
